test/integration/helpers.bash

```diff
@@ -37,17 +37,28 @@ function wait_until_reachable() {
 }
 
 # Start the swarm manager in background.
-function start_manager() {
-	${SWARM_ROOT}/swarm manage -H $SWARM_HOST "$@" `join , ${HOSTS[@]}` &
+function swarm_manage() {
+	${SWARM_ROOT}/swarm manage -H $SWARM_HOST "$@" &
 	SWARM_PID=$!
 	wait_until_reachable $SWARM_HOST
 }
 
+function swarm_join() {
+	for h in ${HOSTS[@]}; do
+		echo "Swarm join $h $@"
+		${SWARM_ROOT}/swarm join --addr=$h "$@" &
+	done
+}
+
 # Stops the manager.
 function stop_manager() {
 	kill $SWARM_PID
 }
 
+function stop_swarm_agents() {
+	kill `pidof swarm`
```

Comment:
This is a bit dangerous since it will kill everything - the agents, the manager, and other swarms running on the host.

What about instead doing something like keeping an array of `PIDs` and then killing them here?

Similar to what we do with IDs `start_docker`  / `stop_docker`?